h/static/scripts/controllers/share-widget-controller.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+const Controller = require('../base/controller');
+
+const CONFIG_ATTR = 'share-widget-config';
+const TRIGGER_SELECTOR = `[${CONFIG_ATTR}]`;
+const WIDGET_SELECTOR = '.js-share-widget-owner';
+const TARGET_HREF_ATTR = 'share-target-href';
+const TARGET_HREF_SELECTOR = `[${TARGET_HREF_ATTR}]`;
+const CLIPBOARD_INPUT_SELECTOR = '.js-share-widget-clipboard';
+const PRIVATE_MSG_SELECTOR = '.js-share-widget-msg-private';
+const GROUP_MSG_SELECTOR = '.js-share-widget-msg-group';
+
+const ARROW_PADDING_RIGHT =  16;
+const ARROW_PADDING_BOTTOM =  5;
+
+let shareWidgetAttached = false;
+
+const getOffset = (el) => {
+  el = el.getBoundingClientRect();
+  return {
+    // adjust for top left of the document
+    left: el.left + window.scrollX,
+    top: el.top + window.scrollY,
```

Comment:
I forgot this when I looked before, but `window.scrollX` and `window.scrollY` are not available in IE 10/11. Changing this to `window.pageXOffset` and `window.pageYOffset` respectively works in IE 11 and newer browsers.